cypress/e2e/models/migration/applicationinventory/application.ts

```diff
@@ -187,10 +185,6 @@ export class Application {
             if (this.sourceRepo) this.fillSourceModeFields();
             if (this.group) this.fillBinaryModeFields();
             submitForm();
-            checkSuccessAlert(
-                commonView.successAlertMessage,
-                `application ${this.name} was successfully saved.`
-            );
```

Comment:
Thanks for removing this from the model! For sure this will prevent many tests from failing!